pkg/ocm/addons.go

```diff
@@ -17,10 +17,22 @@ limitations under the License.
 package ocm
 
 import (
+	"fmt"
+
 	amsv1 "github.com/openshift-online/ocm-sdk-go/accountsmgmt/v1"
 	cmv1 "github.com/openshift-online/ocm-sdk-go/clustersmgmt/v1"
 )
 
+type AddOnBilling struct {
+	BillingModel     string
+	BillingAccountID string
+}
+
+var BillingModels = map[string]cmv1.BillingModel{
+	string(amsv1.BillingModelMarketplace): cmv1.BillingModelMarketplace,
+	string(amsv1.BillingModelStandard):    cmv1.BillingModelStandard,
+}
```

Comment:
Why do we need to translate these instead of using the ams constants directly?

Reply:
For our SDK :see_no_evil: we have our own billing models defined, these consts are duplicated between cs and ams :see_no_evil: and ams has no way to fetch a list of valid billing models, so we are kind of stuck with this duplication.